<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardInfo</class>
 <widget class="QWidget" name="CardInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CardInfo{
	background-color:  rgb(33, 33, 33);
	
}
QLabel{
	color: rgb(189, 189, 189);
}
#cardInfoFrame{
	border: 2px solid rgb(189, 189, 189);
	border-radius:12px;

}
#editButton, #deleteButton {
	padding:5px;
	border:None;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="cardInfoFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>210</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../resources/qrcs/addPerson.qrc">:/images/images/Person.jpg</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="birthdayLabel">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Birthday</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="phoneLabel">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Phone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="emailLabel">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="addressLabel">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="nameInfo">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Unkown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="birthdayInfo">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Unkown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="phoneInfo">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Unkown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="emailInfo">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Unkown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="addressInfo">
              <property name="font">
               <font>
                <family>Inter</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Unkown</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="status">
          <property name="font">
           <font>
            <family>Inter</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="editButton">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/qrcs/addPerson.qrc">
            <normaloff>:/icons/Icons/pencil.png</normaloff>:/icons/Icons/pencil.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/qrcs/addPerson.qrc">
            <normaloff>:/icons/Icons/trash.png</normaloff>:/icons/Icons/trash.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/qrcs/addPerson.qrc"/>
 </resources>
 <connections/>
</ui>
